evap/staff/views.py

```diff
@@ -1535,6 +1535,11 @@ def evaluation_textanswers_update_publish(request):
         answer.hide()
     elif action == "unreview":
         answer.unreview()
+    elif action == "textanswer_edit":
+        url = reverse(
+            "staff:evaluation_textanswer_edit", args=[evaluation.course.semester.id, evaluation_id, textanswer_id]
+        )
+        return HttpResponse(url)
```

Comment:
I wonder if we should `raise PermissionDenied` (or `SuspiciousOperation`) if the requests user is not a manger to not leak ids. It is suspicious after all :thinking: @He3lixxx what do you think?

Reply:
This view is accessible to reviewers only. Reviewers can see all textanswers, so they would have all textanswer IDs anyway(?).

Thinking about this -- this view will give a 500 (in 1520 and 1528) if the provided textanswer id or evaluation id does not exist, instead of giving 404. We might want to fix that.

Reply:
You are right, leaking ids is not a problem :+1:. Requesting this view with `action == "textanswer_edit"` as a reviewer is still, excuse my language, sus

Reply:
yup, definitely sus, but since there's nothing to be gained here, I wouldn't put any additional access control.